diff --git a/wp-includes/post.php b/wp-includes/post.php
index 93bdb0a..294b7e7 100644
--- a/wp-includes/post.php
+++ b/wp-includes/post.php
@@ -1037,6 +1037,8 @@ function register_post_type($post_type, $args = array()) {
                register_taxonomy_for_object_type( $taxonomy, $post_type );
        }
 
+       do_action( 'registered_post_type', $post_type, $args );
+
        return $args;
 }
 
