diff --git a/wp-includes/taxonomy.php b/wp-includes/taxonomy.php
index e231a6c..0d981dc 100644
--- a/wp-includes/taxonomy.php
+++ b/wp-includes/taxonomy.php
@@ -363,6 +363,8 @@ function register_taxonomy( $taxonomy, $object_type, $args = array() ) {
 
        // register callback handling for metabox
        add_filter('wp_ajax_add-' . $taxonomy, '_wp_ajax_add_hierarchical_term');
+
+       do_action( 'registered_taxonomy', $taxonomy, $args );
 }
 
 /**
